cmd/shortener/config/config.go

```diff
@@ -108,10 +116,29 @@ func (c *Config) validateSorageFile(file string) error {
 	return nil
 }
 
+func (c *Config) validateDatabaseDSN(dsn string) error {
+	conf, err := pgx.ParseConfig(dsn)
+	if err != nil {
+		return fmt.Errorf("%s: %w", errorValidateDatabaseDSN, err)
```

Comment:
См. комментарий выше, но просто для информации: при таком форматировании ошибок можно использовать `%w` несколько раз. Тогда при помощи `errors.Is()`/`errors.As()` можно будет проверять ошибку на любой из обернутых типов:


```suggestion
		return fmt.Errorf("%w: %w", errorValidateDatabaseDSN, err)
```

Reply:
Принял